environment/src/main/java/com/sequenceiq/environment/environment/service/freeipa/FreeIpaService.java

```diff
@@ -164,6 +164,30 @@ public Optional<OperationView> getFreeIpaOperation(String environmentCrn, boolea
         }
     }
 
+    public OperationStatus getOperationStatus(String operationId) {
+        String accountId = ThreadBasedUserCrnProvider.getAccountId();
```

Comment:
I would add a debug log here before calling the operation endpoint.

Reply:
ok